ui/moviestrip.reel/moviestrip.js

```diff
@@ -6,6 +6,8 @@ exports.Moviestrip = Component.specialize({
     constructor: {
         value: function Moviestrip () {
             this.super();
+
+            this.application.addEventListener("keydown", this, false);
```

Comment:
@cdebost Thanks, but maybe we could use our KeyComposer here :). Please, find  an exemple here: https://github.com/montagejs/montage/blob/master/ui/base/abstract-number-field.js#L59